frontend/src/components/UserTables/UnscheduledTable.tsx

```diff
@@ -18,59 +18,84 @@ function renderTableHeader() {
 }
 
 type TableProps = {
-  drivers: Driver[]
-}
+  drivers: Driver[];
+};
 const Table = ({ drivers }: TableProps) => {
   const [openModal, setOpenModal] = useState(-1);
-  const passengers = [
-    { startTime: '8:20am', endTime: '8:40am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
-    { startTime: '8:30am', endTime: '8:50am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
-    { startTime: '9:10am', endTime: '9:40am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
-    { startTime: '9:30am', endTime: '9:50am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
-    { startTime: '10:10am', endTime: '10:30am', name: 'Rose Lisborn', pickupLocation: 'Eddygate', pickupTag: 'Ctown', dropoffLocation: 'Hollister Hall', dropoffTag: 'West', needs: 'Crutches' },
-  ];
+  const [rides, setRides] = useState<RealRide[]>([]);
+
+  const compRides = (a: RealRide, b: RealRide) => {
+    const x = new Date(a.startTime);
+    const y = new Date(b.startTime);
+    if (x < y) return -1;
+    if (x > y) return 1;
+    return 0;
+  };
 
-  function renderTableData(allPassengers: Passenger[]) {
-    let currentTime = '';
+  const getUnscheduledRides = () => {
+    fetch('/rides?type=unscheduled')
```

Comment:
Clarify with designers whether this should only be showing unscheduled rides for the current day. Because right now we do not show the dates, only the times and if a later date has an earlier time than an earlier date, the table appears to be out of order. If the designers only want the unscheduled rides for "today", the endpoint should be able to do that for you.

Reply:
Good catch. I'm very sure the home page only displays information for "today".